tools/testing/target_determination/heuristics/historical_edited_files.py

```diff
@@ -0,0 +1,26 @@
+from typing import Any, Dict, List
```

Comment:
Could you leave a detailed comment explaining the idea behind what this heuristic is supposed to do?

We should really do this for all our heuristics